WordPressKit/WordPressKit/ActivityServiceRemote.swift

```diff
@@ -52,6 +52,44 @@ public class ActivityServiceRemote: ServiceRemoteWordPressComREST {
                                 })
     }
 
+    /// Retrieves the site current rewind state.
+    ///
+    /// - Parameters:
+    ///     - siteID: The target site's ID.
+    ///
+    /// - Returns: The current rewind status for the site.
+    ///
+    public func getRewindStatus(_ siteID: Int,
+                                success: @escaping (RewindStatus) -> Void,
+                                failure: @escaping (Error) -> Void) {
+        let endpoint = "sites/\(siteID)/rewind"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
+        let locale = WordPressComLanguageDatabase().deviceLanguage.slug
+        let parameters: [String: AnyObject] = [
```

Comment:
I think if you declared it as `[String: AnyObject]?` here, you could save the cast when passing it in, but that's a super minor nitpick.

Reply:
Thank you for spotting that, fixed now.